vcsh

```diff
@@ -20,15 +20,15 @@
 # version of vcsh; the master branch is supposed to be clean at all times
 # so you can most likely just use it nonetheless
 VERSION='1.20141026'
-SELF=$(basename $0)
+SELF="$(basename "$0")"
```

Comment:
This quoting seems off.

Reply:
Nope, for the shell expressions `${ }` and `$( )` it understands interior quotes. This can be confusing since it does not do this elsewhere (looking at you backticks) but the parens and curly braces give the parser more to work with.

Reply:
I agree it *seems* off, but have also been down the rabbit trail and know it to be correct. It would be pretty vanishingly rare to have a space in the name, but this is technically correct. I'm fine with being a bit pedantic to appease the tooling.